lib/bootloader_pvm.pm

```diff
@@ -117,10 +117,12 @@ sub prepare_pvm_installation {
 
     # Delete partition table before starting installation
     select_console('install-shell');
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
-        script_run("wipefs -a /dev/$disk");
-        # For cryptlvm+activate existing scenario, create empty enrypted partition
-        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+
+
+    my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
+    for my $d (split('\n', $disks)) {
+        script_run "wipefs -a $d";
+        create_encrypted_part("$d") if get_var('ENCRYPT_ACTIVATE_EXISTING');
```

Comment:
Hi, this caused s little issue here https://openqa.suse.de/tests/3909752#step/bootloader_start/36 as then in create_encrypted_part we have  `assert_script_run "parted -s /dev/$disk mklabel gpt";`